components/server/src/test/mocks/storage-client-mock.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { injectable } from "inversify";
+import { StorageClient } from "../../storage/storage-client";
+
+@injectable()
+export class StorageClientMock implements StorageClient {
+    async deleteUserContent(ownerId: string): Promise<void> {
+        // do nothing
```

Comment:
This doesn't throw because it is called during user deletion. But is not explicitly tested so no reason to add some sort of white box testing